archive/introduce-vector-db(kor).md

```diff
@@ -2,14 +2,21 @@
 
 ## 검색 시스템에서 벡터 검색이 왜 중요한가?
```

Comment:
Could you add a reference of this contents? It will help us get better confidence when we show this content to others, from the view of reliablity of contents.